app/utils/plan_utils.rb

```diff
@@ -9,10 +9,13 @@ module PlanUtils
   PRO = 2
   GROWTH = 3
   SCALE = 4
+  HOLD = 5
+  OS = 99999
```

Comment:
These numbers don't make a whole lot sense because comparing in number order is now broken (hold > scale). We should drop the number semantics and store these as symbols instead.